instrumentation/jsp-2.3/src/test/groovy/JSPInstrumentationBasicTests.groovy

```diff
@@ -282,6 +282,18 @@ class JSPInstrumentationBasicTests extends AgentTestRunner {
           operationName expectedOperationName()
           spanKind SERVER
           errored true
+          event(0) {
+            eventName(SemanticAttributes.EXCEPTION_EVENT_NAME)
```

Comment:
Why not usual `errorEvent`?

Reply:
The assertions are more complex not just matching